src/microbe_stage/editor/MicrobeEditor.cs

```diff
@@ -463,10 +463,11 @@ public void OnFinishEditing()
 
         // It is easiest to just replace all
         editedSpecies.Organelles.Clear();
-
+        var centerOfMass = editedMicrobeOrganelles.CenterOfMass;
         foreach (var organelle in editedMicrobeOrganelles.Organelles)
```

Comment:
I'd like a blank line before this foreach

Reply:
You added the blank line Kasterisk wanted, I'd still like to separate the foreach block from the assignment on the previous line.

Reply:
That my previous comment was the review I wanted to do

Reply:
Sorry, didn't read accurately enough. I'll do that tomorrow